src/puppetlabs/trapperkeeper/bootstrap.clj

```diff
@@ -97,27 +99,28 @@
                            (string/join "\n" config-files)))
         config-files))))
 
-(schema/defn ^:private config-from-cwd :- (schema/maybe BootstrapFiles)
+(schema/defn ^:private config-from-cwd :- (schema/maybe [schema/Str])
   "Check to see if there is a bootstrap config file in the current working
   directory;  if so, return it."
   []
   (let [config-file (-> bootstrap-config-file-name
                         (io/file)
                         (.getAbsoluteFile))]
     (when (.exists config-file)
-      (log/debug (str "Loading bootstrap config from current working directory: '"
-                      (.getAbsolutePath config-file) "'"))
-      [config-file])))
+      (let [config-file-path (.getAbsolutePath config-file)]
+        (log/debug (str "Loading bootstrap config from current working directory: '"
+                        config-file-path "'"))
+        [config-file-path]))))
 
-(schema/defn ^:private config-from-classpath :- (schema/maybe BootstrapFiles)
+(schema/defn ^:private config-from-classpath :- [(schema/maybe schema/Str)]
```

Comment:
I think this was supposed to be `(schema/maybe [schema/Str])`, which means your previous comment applies about it not being necessary. Should probably just be `[schema/Str]`